src/parallel_ops/fused_parallel_op.cc

```diff
@@ -50,7 +50,11 @@ bool FusedParallelOpParams::is_valid(ParallelTensorShape const &input) const {
 
 FusedParallelOpParams FusedParallelOp::get_params() const {
   FusedParallelOpParams params;
-  params.parallel_ops = this->parallel_ops;
+  std::vector<ParallelOpInfo> vec;
+  for (ParallelOpInfo op : this->parallel_ops) {
+    vec.push_back(op);
+  }
+  params.parallel_ops = vec;
```

Comment:
```suggestion
  params.parallel_ops = this->parallel_ops;
```
I think the additional copy is unnecessary